### PR TITLE
CCM Release notes for Feb

### PR DESCRIPTION
**Pull request by charanya-j**
# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the maintainers. 

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [ ] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

File: release-notes/cloud-cost-management.md [hunk 1]
```diff
@@ -13,6 +13,30 @@ Harness deploys updates progressively to different Harness SaaS clusters. You ca
...
+
+  This release does not include new features.
+
+### Early access
+
+  This release does not include early access features.
+
+### Fixed issues
+
+* Hourly data on the Perspectives page was showing an incorrect billing amount for many accounts. CloudFunction could not delete the existing records but continued ingesting a new entry in clusterDataHourly in BigQuery. (CCM-10711)
```
Comment (SmritiSatya) [inline release-notes/cloud-cost-management.md:28]: Hourly data on the **Perspectives** page previously showed an incorrect billing amount for multiple accounts. CloudFunction was unable to delete the existing records but continued ingesting new entries in clusterDataHourly in BigQuery. (CCM-10711)

File: release-notes/cloud-cost-management.md [hunk 2]
```diff
@@ -13,6 +13,30 @@ Harness deploys updates progressively to different Harness SaaS clusters. You ca
...
+
+  This issue is fixed. Now, the Instance_Billing_Hourly job execution is limited to 5 times per minute to avoid CloudFunction failure.
+
+* The total costs displayed on the Overview page and the license-util page/API were incorrect in accounts with at least one Azure connector. (CCM-10678)
+
```
Comment (SmritiSatya) [inline release-notes/cloud-cost-management.md:33]: * The total costs displayed on the **Overview** page and the license-util page (or API) were incorrect in accounts with at least one Azure connector. (CCM-10678)

Comment (charanya-j) [reply release-notes/cloud-cost-management.md:33]: No, it means in Harness accounts. Thank you, Smriti.

File: release-notes/cloud-cost-management.md (hunk 1 above)
Comment (charanya-j) [reply release-notes/cloud-cost-management.md:28]: Accepted.

**Review by SmritiSatya — APPROVED** [review]

**Review by SmritiSatya — APPROVED** [review]

**Pull request merged**